config/cypress.config.js

```diff
@@ -192,6 +192,7 @@ const cypressConfig = {
     baseUrl: 'http://localhost:3001',
     specPattern: 'src/**/tests/**/*.cypress.spec.js?(x)',
     supportFile: 'src/platform/testing/e2e/cypress/support/index.js',
+    experimentalSessionAndOrigin: true,
```

Comment:
I'm definitely in favor of clearing session storage before every test. It did occur to me, we could run this by QA people to make sure they're aware of the change